packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -80,6 +80,13 @@ module.exports = (themeOptions = {}) => {
         },
       },
       // Assets (e.g. images) used from the markdown pages
+      {
+        resolve: 'gatsby-source-filesystem',
+        options: {
+          name: 'imagesReleases',
+          path: path.resolve(`./src/images/releases`),
```

Comment:
I don't think we do us a performance favor by having two graphql File nodes for every images when "mounting" the same folder twice (once directly and once as part of `/images`).  

If we want to really fix this we would have to move the release notes images into a completely different conventional place, which I think it's too late for now.

Reply:
We can add the `ignore` option for the `/images` source location to ignore the `releases` folder.

I would try to be careful on loading only the necessary things, as we already do that in a lot of places and I feel like it's easy to get it out of hand if we don't look out for those things.

I mean, this is anyway a workaround and I hope we can find a better solution for the release notes list.

Reply:
Ignoring the /images/releases folder sounds like a good thing to me, but I can't fully see the impact yet. 

I suggest to first check with the @commercetools/docs-lectors (tech writers) whether doing any sort of technical separation of images in release notes and other content is a good idea at all.   If it's intended that any image can be used anywhere then we're doing better in not hardwiring any assumptions here.